README.md

```diff
@@ -25,6 +25,7 @@ jobs:
       - uses: JuliaRegistries/TagBot@v1
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
+          ssh: ${{ secrets.DOCUMENTER_KEY }}    # recommended if you're using Documenter to build and host documentation
```

Comment:
```suggestion
          ssh: ${{ secrets.DOCUMENTER_KEY }}
```

Since it's already explained later in the readme, people can go look for that if they're wondering what this is for IMO.